Assets/Mirror/Components/NetworkAnimator.cs

```diff
@@ -492,17 +491,15 @@ void CmdOnAnimationResetTriggerServerMessage(int hash)
         [ClientRpc]
         void RpcOnAnimationClientMessage(int stateHash, float normalizedTime, int layerId, byte[] parameters)
         {
-            NetworkReader networkReader = NetworkReaderPool.GetReader(parameters);
-            HandleAnimMsg(stateHash, normalizedTime, layerId, networkReader);
-            NetworkReaderPool.Recycle(networkReader);
+            using (NetworkReader networkReader = NetworkReaderPool.GetReader(parameters))
+                HandleAnimMsg(stateHash, normalizedTime, layerId, networkReader);
```

Comment:
please use {} here too, even if it's just one line. we do that (almost) everywhere else too

Reply:
Actually, no, we're not at all consistent on that, and there was a PR months ago that removed braces on a bunch of one-liners as I recall.

Reply:
Confirmed - a whole bunch of commits in late Feb. & early March 2019